packages/communication-react/review/communication-react.api.md

```diff
@@ -449,6 +454,11 @@ export type CallParticipantListParticipant = ParticipantListParticipant & {
     isSpeaking?: boolean;
 };
 
+// @beta
+export type CallParticipantLocator = {
+    participantIDs: string[];
```

Comment:
Currently we support property strip like this:
export type CallParticipantLocator = {
   /* @conditional-compile-remove-from(stable) TEAMS_ADHOC_CALLING */
    participantIDs: string[];
}

This could be a work around to not breaking the current compiling system

Reply:
There shouldn't be anything broken, here is the stable API file
![image](https://user-images.githubusercontent.com/2684369/153058896-4789686e-06ce-433e-a4ed-e3015ccb48f8.png)